pandas/io/json.py

```diff
@@ -197,7 +204,29 @@ def read_json(path_or_buf=None, orient=None, typ='frame', dtype=True,
 
     Returns
     -------
-    result : Series or DataFrame
+    result : Series or DataFrame, depending on the value of ``typ``.
+
+    Examples
+    --------
+
+    >>> df = pd.DataFrame([['a', 'b'], ['c', 'd']],
+                              index=['row 1', 'row 2'],
+                              columns=['col 1', 'col 2'])
+    >>> print df
+          col 1 col 2
+    row 1     a     b
+    row 2     c     d
+    >>> for orient in ['split', 'records', 'index']:
+            str = df.to_json(orient=orient)
+            print "'{}': '{}'".format(orient, str)
+            pd.read_json(str, orient=orient)
```

Comment:
What would you think of the following examples?  We're trying to document `pd.read_json()`, but `df.to_json()` is along for the ride as a convenient source of well-formatted JSON strings. 

The results are a little artificial in that I had to reformat the output of `df.to_json(orient='split')` to avoid the flake8-imposed constraint on  line length.

I also used '\_' to retrieve previous results, but that syntax is not available in ipython when the prompt is '>>> ', as that indicates previous result caching is turned off.  I think using '\_' makes the examples a lot easier to understand, but they won't work if pasted into `%doctest_mode`

![screen shot 2016-09-23 at 3 39 30 pm](https://cloud.githubusercontent.com/assets/295428/18803346/0eabf966-81a4-11e6-8361-6ae3dfde4fdd.png)